src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,22 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def get_styled_command(raw_command):
```

Comment:
We can call this function `highlight_command`, just as `pygments` does in Knack:

https://github.com/microsoft/knack/blob/4cb4c2e0516f760b08defa8b9afacc004f7e4fac/knack/output.py#L44-L46

```py
def format_json_color(obj):
    from pygments import highlight, lexers, formatters
    return highlight(format_json(obj), lexers.JsonLexer(), formatters.TerminalFormatter())  # pylint: disable=no-member
```

Reply:
I agree with jiasli. 

Moreover, as far as I know that the UX design from Lomis adds the `<>` to wrap the parameter values. Such as 
```
az webapp log config --name <WebappName> --resource-group <ResourceGroup> --detailed-error-messages <True or False>
```

Because only the color of each part is returned here, and the style content of text is not processed, I think the function of the style content of text processing also needs another separate and general method. *(That method might call this method to get the color and then modify the content style of text)*
So the naming of the method should be more accurate to distinguish the two methods with different processing degrees.

Reply:
Let's move this discussion to #16349 which is solely for the highlighting functionality.